src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/FlagsEnumTests/JsonStringEnumFlagsRegressionTests.cs

```diff
@@ -0,0 +1,90 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text.Json.Serialization;
+using Xunit;
+
+namespace System.Text.Json.Tests.Serialization.FlagsEnumTests
```

Comment:
Please incorporate the new tests into the existing test suite: https://github.com/dotnet/runtime/blob/main/src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/EnumConverterTests.cs